### PR TITLE
Resize without exception

### PR DESCRIPTION
**Pull request by pmelchior**
uses proxmin 0.5.3 to change the box sizes on the fly.  Only important change for scarlet is that optimization parameters **cannot** be recreated, they need to be modified: https://github.com/fred3m/scarlet/compare/resize-mse?expand=1#diff-ef3b811b0e22d8a2268b81a55665ffd8R304

### CODE REVIEW

File: scarlet/blend.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def fit(self, steps=200, e_rel=1e-2):
             self.it = 0
             self._model_it = -1
             self.converged = False
+            self.mse = []
```
[the remark this answers is not in the document]
Comment (pmelchior) [reply scarlet/blend.py]: Mean Squared Error, pretty common term

[the remark this answers is not in the document]
Comment (pmelchior) [reply scarlet/blend.py]: A more relevant concern is that we have a number of those undocumented class members: it, converged, mse, maybe more. What's the best way to expose them in the docs?

File: scarlet/blend.py [hunk 2]
```diff
@@ -329,7 +333,9 @@ def _prox_f(self, X, step, Xs=None, j=None):
                 grad = np.einsum('...ij,...ij', self._diff, self._models[k])
 
                 # apply per component prox projection and save in component
-                X = self.components[k].sed =  self.components[k].constraints.prox_sed(X - step*grad, step)
+                res = self.components[k].constraints.prox_sed(X - step*grad, step)
```
[the remark this answers is not in the document]
Comment (pmelchior) [reply scarlet/blend.py]: 👍

File: scarlet/blend.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (fred3m) [reply scarlet/blend.py]: We are actually doing our docs slightly wrong. Right now we have `Parameters` in `__init__` for all of our classes when it should actually be in the docstring for the class itself. That is also the place to put an `Attributes` section. See `ExampleError` and `ExampleClass` in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html.

We don't have to go back and fix the `__init__` methods right now but if you add `Attributes` to `Blend` and include a definition of `mse` that should be fine.

[the remark this answers is not in the document]
Comment (pmelchior) [reply scarlet/blend.py]: 👍 for `Attributes`.
However, our documentation is consistent with the docstring style:
>  The __init__ method may be documented in either the class level docstring, or as a docstring on the __init__ method itself.

**Pull request merged**